PowerEditor/src/WinControls/AboutDlg/AboutDlg.cpp

```diff
@@ -381,10 +218,181 @@ void DebugInfoDlg::doDialog()
 	if (!isCreated())
 		create(IDD_DEBUGINFOBOX);
 
+	// Refresh the Debug Information.
+	// For example, the command line parameters may have changed since this dialog was last opened during this session.
+	refreshDebugInfo();
+
 	// Adjust the position of AboutBox
 	goToCenter();
 }
 
+void DebugInfoDlg::refreshDebugInfo()
+{

```

Comment:
Just copy _debugInfoStr into a string, then replace $COMMAND_LINE_PLACEHOLDER$ with the dynamic value in this function.
Set the string value into text zone by using `::SetDlgItemText(_hSelf, IDC_DEBUGINFO_EDIT, debugInfoNewStr.c_str());`.